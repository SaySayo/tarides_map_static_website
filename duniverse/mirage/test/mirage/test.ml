open Mirage

let t = kv_ro @-> job

let test_typ () =
  Alcotest.(check string) "pp" (Fmt.to_to_string Mirage.Type.pp t) "(_ -> _)"

let () = Alcotest.run "mirage" [ ("basic", [ ("pp", `Quick, test_typ) ]) ]
