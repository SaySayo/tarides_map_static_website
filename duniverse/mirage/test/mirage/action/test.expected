unix
====


hvt
===

Write to manifest.json (mode: default, purpose: Solo5 application manifest file)
Write to manifest.ml (0 bytes)

